core/controllers/home.py

```diff
@@ -58,7 +58,8 @@ def get(self):
             raise self.PageNotFoundException
 
         job_queued_msec, recent_notifications = (
-            user_jobs_continuous.DashboardRecentUpdatesAggregator.get_recent_notifications(
+            user_jobs_continuous.DashboardRecentUpdatesAggregator.\
```

Comment:
Ditto, but this is harder to solve. Suggest putting a

   # pylint: disable={{warning-name}}

after the line (where {{...}} is replaced with the appropriate warning name).